addons/google_calendar/static/src/views/google_calendar/google_calendar_controller.js

```diff
@@ -19,17 +19,15 @@ patch(AttendeeCalendarController.prototype, "google_calendar_google_calendar_con
             [[this.user.userId]],
         );
         const syncResult = await this.model.syncGoogleCalendar();
-        if (["need_auth", "need_config_from_admin"].includes(syncResult.status)) {
+        if (syncResult.status === "need_auth") {
+            window.location.replace(syncResult.url);
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Window/location#value
window.location.replace won't give you history, window.location.assign will.
I think it would be better. When something is wrong with the configuration it is always annoying to have to enter manually the Odoo URL again instead of going back in history. :-)